### DISCUSSION TITLE
Monaco Editor

### DISCUSSION THREAD

**Discussion by gustavoabel** [superRaytin/react-monaco-editor#409] [General]
do you know if you can use multiple languages ​​in separate code blocks with monaco editor?

**Comment by guilhermeabell** [comment]


**Reply by gustavoabel** [reply]
sai fora mlk

**Comment by indrareddy15** [comment]
Hi, I'm using the Monaco Editor React package for an SQL query editor. It works fine in local, development, and production environments when the production URL is accessed locally. However, when we access the production or development URL from the servers (ACCOPS client), the package does not load because there is no internet access. When internet access is available, the package loads properly.

Is there any help or guidance from the forum on how to make the package load without internet access?
Below is the URL currently being used:

https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/vs/base/worker/workerMain.js
<img width="651" height="156" alt="image" src="https://github.com/user-attachments/assets/c3a87f4c-c66a-49b3-bf00-bd3d5293c6f7" />
<img width="664" height="195" alt="image" src="https://github.com/user-attachments/assets/bd4334bf-ffcb-418f-9a24-b8dd492f94b7" />
<img width="1494" height="378" alt="image" src="https://github.com/user-attachments/assets/07352d41-e96b-4c84-ab33-cf53e78b6e91" />
